Tests/Test_VoltTempMonitorTask.c

```diff
@@ -157,14 +120,13 @@ void Task2(void *p_arg){
 	assertOSError(err);
 
     while(1){
-        OSMutexPend(&WDog_Mutex, 0, OS_OPT_PEND_BLOCKING, NULL, &err);
-        assertOSError(err);
+        RTOS_BPS_MutexPend(&WDog_Mutex, OS_OPT_PEND_BLOCKING);
         WDog_BitMap |= WD_AMPERES;
         WDog_BitMap |= WD_BALANCING;
         OSMutexPost(&WDog_Mutex, OS_OPT_POST_NONE, &err);
```

Comment:
this didn't get changed to RTOS_BPS_MutexPost

Reply:
also these assertOSErrors can be removed due to our handling that in the wrappers